envs/tensorflow-env.yaml

```diff
@@ -22,6 +21,9 @@ packages:
   - feedstock : tensorflow-metadata
   - feedstock : tensorflow-datasets
   - feedstock : protobuf          #[python == '3.8']
+  - feedstock : tensorflow-datasets
+  - feedstock : tensorflow-metadata
```

Comment:
Maybe we should fix this as well here.  datasets and metadata were duplicated here.